lib/rspec/request_describer.rb

```diff
@@ -70,5 +70,11 @@ def self.included(base)
         end
       end
     end
+
+    # @private
+    def self.process_with_kwargs?
+      ActionDispatch::Integration::Session.instance_method(:process) ||
+      ActionDispatch::Integration::Session.instance_method(:process_with_kwargs) rescue false
```

Comment:
I couldn't understand your point clearly though, since this gem does not specify dependency version of actionpack, I used OR condition to keep backward compatibility.